config/otelcol/otelcol-profiling.yml

```
@@ -0,0 +1,29 @@
+# [OpenTelemetry collector configuration](https://opentelemetry.io/docs/collector/configuration/)
+## For ebpf profiling
+receivers:
+  profiling:
+    SamplesPerSecond: 19
```

Heads up - config format has changed and will be the following in a next version:
```suggestion
    samples_per_second: 19
```